Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM centos
+
+RUN yum -y groupinstall 'Development Tools'
+RUN yum -y install openssl-devel
+
+ADD https://static.rust-lang.org/dist/rust-1.29.0-x86_64-unknown-linux-gnu.tar.gz rust.tar.gz
```

Comment:
AFAICT there are no official Rust packages, and as for rustup, I could have curled it down and run it, but I chose not to use it because I want the builds to be as immutable as possible.